webapp/packages/plugin-data-viewer/src/DatabaseDataModel/Actions/ResultSet/ResultSetDataContentAction.ts

```diff
@@ -165,69 +149,81 @@ export class ResultSetDataContentAction extends DatabaseDataAction<any, IDatabas
 
```

Comment:
`async getFileDataUrl(element: IResultSetElementKey) {`
lets make this function private. it is not in use outside of this action scope. moreover, it is easy to mess up with logic:

1. use it outside to get link
2. forget to put cached link into the cache so we can use it
3. have a bug somewhere

Reply:
fixed